custom_components/tuya_local/devices/royalclima_fresh_climate.yaml

```diff
@@ -371,19 +371,14 @@ secondary_entities:
       - id: 103
         type: boolean
```

Comment:
The `name: sensor` which you removed from below needs to be inserted back here, as otherwise this dp is missing a name.

Reply:
don't understand